pkg/cfn/manager/create_tasks.go

```diff
@@ -6,9 +6,9 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
-// CreateTasksForClusterWithNodeGroups defines all tasks required to create a cluster along
+// NewTasksToCreateClusterWithNodeGroups defines all tasks required to create a cluster along
 // with some nodegroups; see CreateAllNodeGroups for how onlyNodeGroupSubset works
-func (c *StackCollection) CreateTasksForClusterWithNodeGroups(onlyNodeGroupSubset sets.String) *TaskTree {
+func (c *StackCollection) NewTasksToCreateClusterWithNodeGroups(onlyNodeGroupSubset sets.String) *TaskTree {
```

Comment:
I think maybe we could  drop the `WithNodeGroups` part. Or do we have code to create a cluster that doesn't create the nodegroup? Or maybe someone would be tempted to create the cluster and then again the nodegroups?